docs/api/tokenize.rst

```diff
@@ -16,14 +16,40 @@ Modules
 .. autoclass:: Tokenizer
    :members:
 
-NEWMM
------
+
+Tokenize Engine
```

Comment:
As we have several engines, we should use `Engines` instead. Moreover, we should use a proper noun as `ADJ` for `Engines`. So, in short, I think the header should be `Tokenization Engines`.

Reply:
Thanks for reviewing, I have changed from "Tokenize Engine" to "Tokenization Engines".